### PR TITLE
Overview - Joey

### PR DESCRIPTION
**Pull request by gallottaj**

### CODE REVIEW

**Comment by gallottaj** [comment]
Please review the new overview page.

File: docs/index.md
Comment (rbwatson) [inline docs/index.md:1]: I added this so your new page will be accessible.

File: docs/overview-joey.md
Comment (rbwatson) [inline docs/overview-joey.md:1]: This looks pretty good.
I'd suggest providing some more guidance around the tutorials to help new readers get started in an organized manner.

**Review by rbwatson — APPROVED** [review]
LGTM

**Pull request merged**